gene/etl/base.py

```diff
@@ -22,3 +23,14 @@ def _transform_data(self, *args, **kwargs):
     @abstractmethod
     def _add_meta(self, *args, **kwargs):
         raise NotImplementedError
+
+    def _get_normalizer_prefixes(self, *args, **kwargs):
```

Comment:
Prob can remove optional params unless you think we'll need them

```suggestion
    def _get_normalizer_prefixes(self):
```